### ISSUE TITLE
Custom 404 page

### ISSUE THREAD

**Issue by Sancus** [thundernest/thunderbird-website#22]
We don't have one, and the https://www.mozilla.org/en-US/thunderbird/404 mozilla one isn't really appropriate as-is.

Pulling in our header/footer and using similar text but linking to the appropriate places(issues on this repo for website) would be sufficient.

**Comment by ryanleesipes** [comment]
Have opened an issue for this on Mozilla's open design repo: https://github.com/mozilla/OpenDesign/issues/195

**Comment by jotasprout** [comment]
What's the status on this? Also asking  at [https://github.com/mozilla/OpenDesign/issues/195](https://github.com/mozilla/OpenDesign/issues/195).

**Comment by ryanleesipes** [comment]
Needs to be implemented. @jotasprout if you have any development experience, feel free to do it. If not, let me know and I'll try and get together a design with the resources we got from Open Design.

**Comment by jotasprout** [comment]
Sure, @ryanleesipes. Now, when you say it needs to be implemented by someone with development experience ... do you mean something beyond just putting it in via html/css? Looking at [mozilla/OpenDesign#195](https://github.com/mozilla/OpenDesign/issues/195), it looks like just placing an image, really. Am I incorrect?

**Comment by ryanleesipes** [comment]
@jotasprout No, I just mean someone with a good sense of design who can actually do it. Mine looks like terrible, but you might have a better eye for it.

**Comment by jotasprout** [comment]
Yes, @ryanleesipes, I'm more than happy to give it a shot.

**Comment by jotasprout** [comment]
I'll get back on this and #36 this weekend.

**Comment by ryanleesipes** [comment]
@jotasprout did you have any luck?

**Comment by jotasprout** [comment]
Was another completely busy weekend but I have today off and am working on personal stuff today so I can get to it. Can we IM or chat or whatever? I have a few questions and commenting on issues like this drives me crazy sometimes.

**Comment by jotasprout** [comment]
Which directory should I pop it in? **Stupid question:** Once it is done, there's an admin who can configure the host/server to use it, correct? I have customized existing 404 pages but never made one from scratch so I don't know if they're like an index page where the server knows to automatically use it by the name or not.

**Comment by Sancus** [comment]
Which page to use for 404 is configured by the web server, so don't worry about that, it's not part of this repo. You can just add it as `404.html` alongside the main `index.html` in `website/`

I suggest posting a screenshot in this thread when you have something as a first step, though.

**Comment by ryanleesipes** [comment]
It's been merged!
